tests/Integration/Behaviour/Features/Context/Domain/AttachmentFeatureContext.php

```diff
@@ -113,7 +113,7 @@ private function uploadDummyFile(string $fileName): string
 
         $destination = _PS_DOWNLOAD_DIR_ . $fileName;
         copy($source, $destination);
-        chmod($destination, 7550);
+        chmod($destination, 6440);
```

Comment:
```suggestion
        chmod($destination, 0644);
```

6440 is not a right mode :thinking: